src/main/scala/temple/generate/server/CreatedByAttribute.scala

```diff
@@ -2,8 +2,38 @@ package temple.generate.server
 
 import temple.ast.AttributeType
 
-case class CreatedByAttribute(
-  inputName: String,
-  name: String,
-  attributeType: AttributeType,
-)
+sealed trait CreatedByAttribute
+
+object CreatedByAttribute {
+
+  /**
+    * None encapsulates the case where no Temple defined createdBy field exists, e.g. in the case of an Auth block
+    */
+  case object None extends CreatedByAttribute
+
+  sealed trait Enumerating extends CreatedByAttribute{
```

Comment:
Please run the autoformatter over everything
```suggestion
  sealed trait Enumerating extends CreatedByAttribute {
```